modules/custom/activity_creator/activity.page.inc

```diff
@@ -53,7 +53,22 @@ function template_preprocess_activity(array &$variables) {
   // 'compact_notification' view mode on the User entity. Note that the
   // 'compact_notification' view mode might not be configured, so remember to
   // always check the theme setting first.
-  $account = $activity->getOwner();
+  $entities = $activity->field_activity_entity->referencedEntities();
```

Comment:
FYI @OH17L @bramtenhove : This needs to be more defensive in the future. If entities don't exist then this is now causing an issue in some installations.

Reply:
Fixed in PR #1228.